lib/rucio/daemons/hermes/hermes.py

```diff
@@ -137,16 +397,15 @@ def submit_to_elastic(
 
     for message in messages:
         text += '{ "index":{ } }\n%s\n' % json.dumps(message, default=default)
-    res = requests.post(endpoint,
-                        data=text,
-                        headers={"Content-Type": "application/json"},
-                        auth=auth)
+    res = requests.post(
+        endpoint, data=text, headers={"Content-Type": "application/json"}, auth=auth
+    )
     return res.status_code
 
 
 def aggregate_to_influx(
     messages: "Iterable[dict[str, Any]]",
-    bin_size: str,
+    bin_size: int,
```

Comment:
I think `bin_size` is expected to be `str`. The only usage I can find is `bin_size="1m"`.

Reply:
```
   - 1 errors with message """Argument of type "Literal['1m']" cannot be assigned to parameter "bin_size" of type "int" in function "aggregate_to_influx"
                               "Literal['1m']" is not assignable to "int"""".
    Candidates: line 679
```